.goreleaser.yml

```diff
@@ -137,33 +140,151 @@ nfpms:
+      - dst: /opt/observiq-otel-collector/plugins/hbase_logs.yaml
+        type: ghost
+      - dst: /opt/observiq-otel-collector/plugins/iis_logs.yaml
+        type: ghost
+      - dst: /opt/observiq-otel-collector/plugins/ingress_nginx_logs.yaml
+        type: ghost
+      - dst: /opt/observiq-otel-collector/plugins/jboss_logs.yaml
+        type: ghost
+      - dst: /opt/observiq-otel-collector/plugins/json_logs.yaml
+        type: ghost
+      - dst: /opt/observiq-otel-collector/plugins/kafka_logs.yaml
+        type: ghost
+      - dst: /opt/observiq-otel-collector/plugins/kubelet_logs.yaml
+        type: ghost
+      - dst: /opt/observiq-otel-collector/plugins/macos_logs.yaml
+        type: ghost
+      - dst: /opt/observiq-otel-collector/plugins/mongodb_logs.yaml
+        type: ghost
+      - dst: /opt/observiq-otel-collector/plugins/mysql_logs.yaml
+        type: ghost
+      - dst: /opt/observiq-otel-collector/plugins/nginx_logs.yaml
+        type: ghost
+      - dst: /opt/observiq-otel-collector/plugins/oracle_database_logs.yaml
+        type: ghost
+      - dst: /opt/observiq-otel-collector/plugins/oracledb_metrics.yaml
+        type: ghost
+      - dst: /opt/observiq-otel-collector/plugins/pgbouncer_logs.yaml
+        type: ghost
+      - dst: /opt/observiq-otel-collector/plugins/postgresql_logs.yaml
+        type: ghost
+      - dst: /opt/observiq-otel-collector/plugins/rabbitmq_logs.yaml
+        type: ghost
+      - dst: /opt/observiq-otel-collector/plugins/redis_logs.yaml
+        type: ghost
+      - dst: /opt/observiq-otel-collector/plugins/sap_hana_logs.yaml
+        type: ghost
+      - dst: /opt/observiq-otel-collector/plugins/solr_logs.yaml
+        type: ghost
+      - dst: /opt/observiq-otel-collector/plugins/sql_server_logs.yaml
+        type: ghost
+      - dst: /opt/observiq-otel-collector/plugins/syslog_logs.yaml
+        type: ghost
+      - dst: /opt/observiq-otel-collector/plugins/tcp_logs.yaml
+        type: ghost
+      - dst: /opt/observiq-otel-collector/plugins/tomcat_logs.yaml
+        type: ghost
+      - dst: /opt/observiq-otel-collector/plugins/ubiquiti_logs.yaml
+        type: ghost
+      - dst: /opt/observiq-otel-collector/plugins/udp_logs.yaml
+        type: ghost
+      - dst: /opt/observiq-otel-collector/plugins/vcenter_logs.yaml
+        type: ghost
+      - dst: /opt/observiq-otel-collector/plugins/w3c_logs.yaml
+        type: ghost
+      - dst: /opt/observiq-otel-collector/plugins/wildfly_logs.yaml
+        type: ghost
+      - dst: /opt/observiq-otel-collector/plugins/windows_dhcp.yaml
+        type: ghost
+      - dst: /opt/observiq-otel-collector/plugins/zookeeper_logs.yaml
+        type: ghost
+      - dst: /usr/share/observiq-otel-collector/stage/observiq-otel-collector
         type: dir
         file_info:
-          mode: 0755
-          owner: observiq-otel-collector
-          group: observiq-otel-collector
+          mode: 0750
+          owner: root
+          group: root
```

Comment:
The staging dir `/usr/share/observiq-otel-collector/stage/observiq-otel-collector` is not meant to be readable by all users, therefore is owned by root. The final install location will be owned by `observiq-otel-collector` user, just like before. See the changes to the package postinstall script.